Rakefile

```diff
@@ -27,6 +22,7 @@ end
 namespace :test do
   desc 'Run all development tests located in the test directory'
   Rake::TestTask.new :dev do |task|
+    task.options = RakeArguments.task_options
```

Comment:
so that's where the task_options name comes from.

There's probably a standard name for options after `--` that would be better. (see also name issue above.)